### PR TITLE
feat: show user's current time in the contributions page

### PR DESCRIPTION
**Pull request by srl295**
image

### CODE REVIEW

File: shared/users.ts [hunk 1]
```diff
@@ -1,20 +1,20 @@
 // github ids / community ids / avatars currently unused
-export const userIds: {[index:string]: {community:string, avatar?: string}} = {
-  "darcywong00": { community: "darcy", avatar: 'bg+DW.png'},
+export const userIds: {[index:string]: {community:string, avatar?: string, tz?: string}} = {
+  "darcywong00": { community: "darcy", avatar: 'bg+DW.png', tz: 'Asia/Bangkok'},
   "ermshiperete": { community: "EberhardBeilharz" },
```
Comment (mcdurdin) [inline shared/users.ts:4]:
```suggestion
  "ermshiperete": { community: "EberhardBeilharz", tz: 'Europe/Berlin' },
```

File: shared/users.ts [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-  "jahorton": { community: "joshua_horton", avatar: 'bg+JH.png'},
-  "MakaraSok": { community: "makara", avatar: 'bg+MS.png'},
-  "mcdurdin": { community: "Marc", avatar: 'bg+MD.png'},
+  "jahorton": { community: "joshua_horton", avatar: 'bg+JH.png', tz: 'Asia/Phnom_Penh'},
+  "MakaraSok": { community: "makara", avatar: 'bg+MS.png', tz: 'Asia/Phnom_Penh'},
+  "mcdurdin": { community: "Marc", avatar: 'bg+MD.png', tz: 'Asia/Phnom_Penh'},
   "rc-swag": { community: "ross", avatar: 'bg+RC.png'},
```
Comment (mcdurdin) [inline shared/users.ts:8]:
```suggestion
  "rc-swag": { community: "ross", avatar: 'bg+RC.png', tz: 'Australia/Brisbane'},
```

File: shared/users.ts [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
   "SabineSIL": { community: "", avatar: 'bg+SAB.png'},
```
Comment (mcdurdin) [inline shared/users.ts:9]:
```suggestion
  "SabineSIL": { community: "", avatar: 'bg+SAB.png', tz: 'Europe/Berlin'},
```

File: shared/users.ts [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
-  "sgschantz": { community: "Shawn", avatar: 'bg+SGS.png'},
-  "srl295": { community: "srl295", avatar: 'bg+SRL.png'},
+  "sgschantz": { community: "Shawn", avatar: 'bg+SGS.png', tz: 'Asia/Phnom_Penh'},
+  "srl295": { community: "srl295", avatar: 'bg+SRL.png', tz: 'America/Chicago'},
   "bharanidharanj": { community: "", avatar: 'bg+BJ.png'},
   "dinakaranr": {community: "" },
```
Comment (mcdurdin) [inline shared/users.ts:13]:
```suggestion
  "bharanidharanj": { community: "", avatar: 'bg+BJ.png', tz: 'Asia/Kolkata'},
  "dinakaranr": {community: "", tz: 'Asia/Kolkata' },
```

**Review by mcdurdin — APPROVED** [review]

**Review by mcdurdin — APPROVED** [review]
LGTM. Won't track my movements though and I don't plan to keep my TZ up to date :grin:

**Comment by srl295** [comment]
and thanks for filling in the rest of the tzs- i was getting to those.

**Comment by srl295** [comment]
> LGTM. Won't track my movements though and I don't plan to keep my TZ up to date 😁

```js
mcdurdin: { tz: fetch("https://durd.in/tz.json") }
```

**Pull request merged**